src/ngraph_deassign_clusters.cc

```diff
@@ -99,6 +99,47 @@ Status DeassignClusters(Graph* graph) {
     }
   }
 
+  //
+  // At this point we have made our final decision about cluster assignment, so
+  // we will log the cluster assignment now.
+  //
+  if (NGRAPH_VLOG_IS_ON(2)) {
+    // Can't quite reuse cluster_map here, unfortunately.
+    std::map<int, std::set<const Node*>> final_cluster_map;
+
+    for (auto node : graph->nodes()) {
+      int cluster_idx;
+      if (!GetNodeCluster(node, &cluster_idx).ok()) {
+        cluster_idx = -1;
+      }
+      final_cluster_map[cluster_idx].insert(node);
+    }
+
+    for (auto kv : final_cluster_map) {
+      int cluster_idx = kv.first;
+      std::set<const Node*>& nodes = kv.second;
+
+      if (cluster_idx == -1) {
+        NGRAPH_VLOG(2) << "FINAL CLUSTER ASSIGNMENT: NO CLUSTER ("
+                       << nodes.size() << " NODES)";
+        NGRAPH_VLOG(2) << "===================================================="
+                          "===============";
+      } else {
+        NGRAPH_VLOG(2) << "FINAL CLUSTER ASSIGNMENT: CLUSTER " << cluster_idx
+                       << " (" << nodes.size() << " NODES)";
+        NGRAPH_VLOG(2) << "===================================================="
+                          "===============";
+      }
+
+      for (auto node : nodes) {
+        NGRAPH_VLOG(2) << node->name() << "[" << node->type_string() << "]";
```

Comment:
Add a prefix such as `NGRAPH_ENCAP: `. Also add a space before the node type so the complete string looks like the following:
```
2018-08-20 14:06:58.295614: I /localdisk/amprocte/tf-di/ngraph-tf/src/ngraph_deassign_clusters.cc:135] NGRAPH_ENCAP: v0/cg/resnet_v17/conv24/conv2d/kernel/Initializer/random_uniform/min [Const]
2018-08-20 14:06:58.295620: I /localdisk/amprocte/tf-di/ngraph-tf/src/ngraph_deassign_clusters.cc:135] NGRAPH_ENCAP: v0/cg/resnet_v113/conv44/conv2d/kernel/Initializer/random_uniform/sub [Const]
2018-08-20 14:06:58.295626: I /localdisk/amprocte/tf-di/ngraph-tf/src/ngraph_deassign_clusters.cc:135] NGRAPH_ENCAP: v0/cg/resnet_v113/conv44/conv2d/kernel/Initializer/random_uniform/mul [Mul]
```
This will help running a `grep` followed by other `awk` kung-fu to separate out the list of op names and their type etc.